Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -238,11 +240,13 @@ public void updateCharity(Charity charity) throws Exception {
   }
   // Function takes modified class and updates database.
   public void updateTag(Tag tag) throws Exception {
-    Entity tagEntity = datastore.get(tag.getId());
-    tagEntity.setProperty(NAME, tag.getName());
-    tagEntity.setProperty(TRENDING_SCORE, tag.getTrendingScoreTag());
-    tagEntity.setProperty(IMGSRC, tag.getImgSrc());
-    datastore.put(tagEntity);
+    Query query = new Query(TAG).setFilter(new FilterPredicate("id", FilterOperator.EQUAL, tag.getId().getId()));
```

Comment:
changes in this file could have been submitted as a separate PR

Reply:
Do we also need to update the corrosponding test file?

Reply:
I believe that is something Toni is looking into.

Reply:
Please fix all related tests to your CL even if they have been authored by someone else.